packages/ag-charts-community/src/chart/series/polar/donutSeries.ts

```diff
@@ -690,8 +690,8 @@ export class DonutSeries extends PolarSeries<PieDonutNodeDatum, AgDonutSeriesOpt
     ) {
         const { fills, strokes, itemStyler } = this.properties;
 
-        const defaultStroke = strokes[datumIndex];
-        const defaultFill = fills[datumIndex];
+        const defaultStroke = strokes[modulus(datumIndex, strokes.length)];
+        const defaultFill = fills[modulus(datumIndex, fills.length)];
```

Comment:
As discussed, the better fix would be to do this in integrated with theme operations.